packages/compass-aggregations/src/utils/modal-descriptions.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Link } from '@mongodb-js/compass-components';
+
+export const runPipelineConfirmationDescription = ({
+  typeOfWrite,
+  stage,
+  ns,
+}: {
+  typeOfWrite: string;
+  stage: { name: string; link: string };
+  ns: string | null;
+}) => {
+  if (!ns) {
```

Comment:
Nit: seems like the only difference is the text after the comma, would probably be easier to update shared parts if it's all the same and only what's different is changing:

```
const nsText = ns ? <>{typeOfWrite} &quot;<b>{ns}</b>&quot;<> : <>that may alter or overwrite a collection<>

// ...

operation, {nsText}. Do you wish to proceed?
```